### ISSUE TITLE
use `data type` instead of `key type` in `index_definition` FT.INFO

### ISSUE THREAD

**Issue by prnam** [RediSearch/RediSearch#2508]
Hey RediSearch Team,
We bumped into an RediSearch issue while updating the RedisInsight 2.0 [here](https://github.com/RedisInsight/RedisInsight/pull/201#pullrequestreview-854590189).  As per Redis.io and [FT.CREATE](https://oss.redis.com/redisearch/Commands/#format) this should be using the term `data_type` and not `key_type`. I kindly request you to review this and help us.

 Please feel free to ask any questions or any additional information required on this issue raised. Happy to answer and  looking forward for fix soon 🙂

**Comment by gkorland** [comment]
you're right `key_type` == `data_type` we'll change the docs to `key_type`

**Comment by prnam** [comment]
No. You should not change doc. You should change the code. 😅
